people.md

```diff
@@ -0,0 +1,29 @@
+# pandas People
+
+## Benevolent Dictator for Life
+
+Wes McKinney is the Benevolent Dictator for Life (BDFL).
+
+## Project Core Developers
+
+- Tom Augspurger
+- Phillip Cloud
+- Andy Hayden
+- Stephan Hoyer
+- Thomas Kluyver
+- Wes McKinney
+- Wouter Overmeire
```

Comment:
are we considering only 'active' developers (for 2015)? e.g. *lots* of people have commit rights, but didn't list everyone here (nor should we). Several don't have commits since 2013/14.